From fe714df9641286e3f005e0021e1c6462fb0b061e Mon Sep 17 00:00:00 2001
From: Trevor Woerner <trevor.woerner@linaro.org>
Date: Fri, 7 Feb 2014 04:07:17 +0100
Subject: [PATCH] qtwebkit: fix QA issue (bad RPATH)

Building qtwebkit causes a QA issue such that QtWebPluginProcess and
QtWebProcess contain bad RPATHs which point into the build location. This fix
adds a patch to not include the rpath.prf which causes this problem.

Signed-off-by: Trevor Woerner <trevor.woerner@linaro.org>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Signed-off-by: Jonathan Liu <net147@gmail.com>
---
 Tools/qmake/mkspecs/features/unix/default_post.prf | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Tools/qmake/mkspecs/features/unix/default_post.prf b/Tools/qmake/mkspecs/features/unix/default_post.prf
index 9c52406..7ece013 100644
--- a/Tools/qmake/mkspecs/features/unix/default_post.prf
+++ b/Tools/qmake/mkspecs/features/unix/default_post.prf
@@ -59,8 +59,6 @@ linux-*g++* {
     }
 }
 
-contains(TEMPLATE, app): CONFIG += rpath
-
 CONFIG(debug, debug|release)|force_debug_info {
     # Make ld not cache the symbol tables of input files in memory to avoid memory exhaustion during the linking phase.
     !force_static_libs_as_shared:config_gnuld: QMAKE_LFLAGS += -Wl,--no-keep-memory
